outputs.tf

```diff
@@ -1,6 +1,7 @@
 output "cluster_id" {
   description = "The name/id of the EKS cluster."
   value       = element(concat(aws_eks_cluster.this.*.id, list("")), 0)
+  depends_on  = [null_resource.wait_for_cluster]
```

Comment:
So that calling plans don't need to recreate the `wait_for_cluster` data source. Doesn't appear to cause any negative effects in my testing

Reply:
Nice. Can you add it as a comment please.

Reply:
👍